substrate/frame/assets/src/lib.rs

```diff
@@ -415,6 +450,15 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
+	/// The asset ID to be used for the next asset creation, if any present. Otherwise, it has no
+	/// effect.
+	///
+	/// The next asset ID can be set using the
+	/// [SetNextAssetId](`migration::next_asset_id::SetNextAssetId`) migration. For the auto-
+	/// incremented model, the [`crate::AutoIncAssetId`] callback can be used.
+	#[pallet::storage]
+	pub type NextAssetId<T: Config<I>, I: 'static = ()> = StorageValue<_, T::AssetId, OptionQuery>;
```

Comment:
There is no way to set this at genesis. Can you please add it to the `GenesisConfig`?

Reply:
I did not touch it to make this non breaking change for most of the users. But no strong opinion. 
You can see it now in `GenesisConfig`.